test/fixture_builder_test.rb

```diff
@@ -83,4 +82,19 @@ def test_absolute_rails_fixtures_path
   def test_fixtures_dir
     assert_match /test\/fixtures$/, FixtureBuilder.configuration.send(:fixtures_dir).to_s
   end
+
+  def test_rebuilding_due_to_differing_file_hashes
+    create_and_blow_away_old_db
+    force_fixture_generation_due_to_differing_file_hashes
```

Comment:
I copied the existing `test_serialization` test above and just changed this line.